config/scootconfig/modules.go

```diff
@@ -129,7 +129,7 @@ func (c *WorkersThriftConfig) Create(
 		di := dialer.NewSimpleDialer(tf, pf)
 		cl, _ := client.NewSimpleClient(di, string(node.Id()))
 		q := runners.NewPollingService(cl, cl, cl, pollingPeriod)
-		return workers.NewServiceWorker(q, timeout, defaultOverhead)
+		return workers.NewServiceWorker(q, taskTimeout, defaultOverhead)
```

Comment:
Not necessarily the PR for it, but: what is the right place to clarify that a task has to reach a PREPARING/RUNNING state before taskTimeout is started? When we use timeout in smoketest, iirc, we expect a task to be canceled regardless of queued status. Should we add a way to specify when timeout starts ticking?